dev/com.ibm.ws.microprofile.faulttolerance.2.1_fat_tck/bnd.bnd

```diff
@@ -19,8 +19,6 @@ src: \
 
```

Comment:
This commit is really a fixup for " Added code to run tck tests for fault tolerance 2.1. " - can you merge the two together.